### PR TITLE
Binance futures auth for User data streams

### PR DESCRIPTION
**Pull request by twmeggs**
### Allows use of USER_DATA streams from Binance USDT futures.

The subscription name is `USER_DATA`, which produces five user-account related streams: account updates, order updates, margin calls, account config changes, and `listenKey` expiration (see [docs](https://binance-docs.github.io/apidocs/futures/en/#user-data-streams)).

This pull request code implements callbacks to handle account updates (`ACCOUNT_UPDATE`) and order updates (`ORDER_INFO`)

- [x] - Tested
- [x] - Changelog updated
- [x] - Tests run and pass
- [x] - Flake8 run and all errors/warnings resolved
- [x] - Contributors file updated (optional)

### CODE REVIEW

**Comment by xiandong79** [comment]
what about a demo example? which has some expected output in its comments.

**Comment by Herklos** [comment]
Nice job @twmeggs, can't wait to use it with cryptofeed

File: cryptofeed/exchange/binance_futures.py [hunk 1]
```diff
@@ -118,5 +118,143 @@ async def message_handler(self, msg: str, conn: AsyncConnection, timestamp: floa
...
+                    "up":"-1423.15600",
+                    "mt":"isolated",
+                    "iw":"6570.42511771",
+                    "ps":"SHORT"
+                }
+              ]
+            }
+        }
+        """
+        await self.callback(ACCOUNT_UPDATE,
```
Comment (bmoscon) [inline cryptofeed/exchange/binance_futures.py:256]: it doesnt look like you actually are returning any data here? Wouldnt you want to return information about the account here too?

File: cryptofeed/feed.py [hunk 2]
```diff
@@ -207,7 +206,6 @@ def connect(self) -> List[Tuple[AsyncConnection, Callable[[None], None], Callabl
     def info(cls) -> dict:
         """
         Return information about the Exchange - what trading symbols are supported, what data channels, etc
-
```
Comment (bmoscon) [inline cryptofeed/feed.py:210]: i like this whitespace

File: cryptofeed/feed.py [hunk 3]
```diff
@@ -257,16 +255,13 @@ def symbol_mapping(cls, symbol_separator='-', refresh=False) -> Dict:
     async def book_callback(self, book: dict, book_type: str, symbol: str, forced: bool, delta: dict, timestamp: float, receipt_timestamp: float):
         """
         Three cases we need to handle here
-
```
Comment (bmoscon) [inline cryptofeed/feed.py:260]: same here, i think its easier to read with the whitespace in the comments

File: cryptofeed/standards.py [hunk 4]
```diff
@@ -1,10 +1,7 @@
 '''
 Copyright (C) 2017-2021  Bryant Moscon - bmoscon@gmail.com
-
 Please see the LICENSE file for the terms and conditions
 associated with this software.
-
```
Comment (bmoscon) [inline cryptofeed/standards.py:6]: unclear why this was removed, please put it back

**Review by bmoscon — CHANGES_REQUESTED** [review]

**Comment by bmoscon** [comment]
@twmeggs overall - looks good. there are many places where you removed blank lines in comments - please undo those changes. There was one other comment about a callback, so let  me know if i missed something there

**Comment by bmoscon** [comment]
@twmeggs sorry, I think this has divered too much for me to fix it. There is also the issue with the account_update callback that does nothing (it just returns a timestamp really). I don't know how to fix that. If you want to resolve these issues or rework this in a new branch, that would be wonderful. If not, that's ok to. I'm going to close this since it is un-mergeable as is. If you want to re-open this or open a new PR later, thats fine too

**Pull request closed without merging**